features/FEATURE_BLE/ble/gap/AdvertisingParameters.h

```diff
@@ -124,6 +124,9 @@ class AdvertisingParameters {
      * A range is provided to the LE subsystem, so it can adjust the advertising
      * interval with other transmission happening on the BLE radio.
      *
+     * @note If CONNECTABLE_UNDIRECTED or CONNECTABLE_DIRECTED advertising is used
```

Comment:
It would make sense to have a constructor that accepts the advertising type and the legacy flag.

Reply:
added